src/components/settings/SafeModules/RemoveModule/steps/ReviewRemoveModule.tsx

```diff
@@ -26,10 +26,10 @@ export const ReviewRemoveModule = ({ data, onSubmit }: { data: RemoveModuleData;
     }
   }, [safeTxError])
 
-  const onFormSubmit = (data: null) => {
+  const onFormSubmit = () => {
```

Comment:
`SignOrExecuteForm` still has `data: null` as the method signature for `onSubmit` but it seems to not cause any issues

Reply:
I've undone these changes. It's probably best we tackle these with the stepper type inference we've been discussing.